src/views/report/ScriptReportDetail/index.tsx

```diff
@@ -209,27 +212,26 @@ function ExecutionDetail({ state }: IObserveProps) {
                                 <Translate msg="script_reports.detail.main.action.go_to_parent_script_detail" />
                             </Button>
                         )}
-                        <Box flex="0 0 auto" marginLeft={72}>
-                            <Button
-                                variant="contained"
-                                color="primary"
-                                size="small"
-                                endIcon={<ChevronRight />}
-                                onClick={() => redirectTo({
-                                    routeKey: ROUTE_KEYS.R_SCRIPT_DETAIL,
-                                    params: {
-                                        name: scriptExecutionData.scriptName,
-                                        version: scriptExecutionData.scriptVersion,
-                                    },
-                                })}
-                                hidden={!checkAuthority(
-                                    SECURITY_PRIVILEGES.S_SCRIPTS_READ,
-                                    scriptExecutionData.securityGroupName,
-                                )}
-                            >
-                                <Translate msg="script_reports.overview.header.redirect_to" />
-                            </Button>
-                        </Box>
+                        <Button
+                            className={classes.redirectButton}
+                            variant="contained"
+                            color="primary"
+                            size="small"
+                            endIcon={<ChevronRight />}
+                            onClick={() => redirectTo({
+                                routeKey: ROUTE_KEYS.R_SCRIPT_DETAIL,
+                                params: {
+                                    name: scriptExecutionData.scriptName,
+                                    version: scriptExecutionData.scriptVersion,
+                                },
+                            })}
+                            hidden={!checkAuthority(
+                                SECURITY_PRIVILEGES.S_SCRIPTS_READ,
+                                scriptExecutionData.securityGroupName,
+                            )}
+                        >
+                            <Translate msg="script_reports.overview.header.redirect_to" />
```

Comment:
To avoid ambiguity, please refactor the following button texts:
- 'Back to parent script' -> 'Back to parent execution'
- 'Go to script details' -> 'Go to script design'
- 'View script details' -> 'View script execution'